crates/node/src/args.rs

```diff
@@ -546,17 +549,18 @@ pub struct L1ProviderArgs {
 /// The arguments for the Beacon provider.
 #[derive(Debug, Default, Clone, clap::Args)]
 pub struct BeaconProviderArgs {
```

Comment:
Should we change the name of these args to 
```suggestion
pub struct BlobProviderArgs {
```